src/backend/expungeservice/endpoints/users.py

```diff
@@ -51,5 +51,24 @@ def post(self):
 
         return jsonify(response_data), 201
 
+    @admin_auth_required
+    def get(self):
```

Comment:
Perhaps we can name this index as it is indexing all users? I think the name get is very generic as it could refer to `get_users`, `get_user`,  `get_user_new`, or `get_user_edit`?

Reply:
This function is called get because it's the GET http request of the /api/users/ endpoint, and the MethodView class in Flask for defining endpoints needs to use the same name. 

I agree it's all a little too generic and needs redesign. I think the present users.GET to get all users seems fine but users.POST to create a new user is confusing. This deserves its own issue to hash out a naming scheme.

Reply:
I see. Should we write down all of our endpoints (URI's), and document how they're going to be routed so that we can use some naming conventions that imply intent?

Reply:
Yes, lets.